instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```diff
@@ -0,0 +1,262 @@
+---
+rules:
+  - bean: org.apache.camel:context=*,type=context,name=*
+    prefix: camel.context.
+    metricAttribute:
+      context: param(context)
+      camelVersion: beanattr(CamelVersion)
+    mapping:
+      ExchangesCompleted:
+        metric: exchange.completed
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges processed successfully since context start-up or the last reset operation.
+      ExchangesFailed:
+        metric: exchange.failed
+        type: counter
+        unit: "{exchanges}"
+        desc: Indicates the total number of exchanges that failed to process since context start-up or the last reset operation.
+      ExchangesInflight:
+        metric: exchange.inflight
+        type: updowncounter
+        unit: "{exchanges}"
+        desc: Indicates the number of exchanges currently transiting the context.
+      ExchangesTotal:
+        metric: exchange.total
+        type: counter
```

Comment:
I don't think we should use `.total`, check out https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#do-not-use-total

maybe `exchange.count`?

Reply:
No, this is a Counter not an UpDownCounter:

> UpDownCounters SHOULD NOT use _total because then they will look like monotonic sums.

Reply:
the next line says:

> Counters SHOULD NOT append `_total` either because then their meaning will be confusing in delta backends.

also check out https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#use-count-instead-of-pluralization-for-updowncounters